File: benchmarks/cross-language-v1/tasks/clean-install-calculator-v1/public/typescript/candidate.ts
// The starter calculator module a fresh `semaprax new <dest> --template
// calculator` scaffold ships with one operation (`add`); this port's
// baseline mirrors that same minimal two-function calculator shape a fresh
// TypeScript project would start from before any of its own logic exists.
export function add(left: number, right: number): number {
  return left + right;
}

export function subtract(left: number, right: number): number {
  return left - right;
}

File: benchmarks/cross-language-v1/tasks/clean-install-calculator-v1/public/rust/candidate.rs
// The starter calculator module a fresh `semaprax new <dest> --template
// calculator` scaffold ships with one operation (`add`); this port's
// baseline mirrors that same minimal two-function calculator shape a fresh
// Rust project would start from before any of its own logic exists.
pub fn add(left: i64, right: i64) -> i64 {
    left + right
}

pub fn subtract(left: i64, right: i64) -> i64 {
    left - right
}
